src/transformers/trainer.py

```diff
@@ -4461,6 +4460,27 @@ def create_accelerator_and_postprocess(self):
             wrapper = "DeepSpeed" if self.is_deepspeed_enabled else "FSDP"
             raise NotImplementedError(f"`{wrapper}` doesn't support `auto_find_batch_size`.")
 
+    def free_memory(self, *models):
```

Comment:
I think the access to the model is also present, one can use `self.model`, `self.deepspeed` and `self.model_wrapped`.

Reply:
Done! (many weeks later)